app.py

```diff
@@ -1078,7 +1087,11 @@ def add_blast_subscription(form=None, customer=None):
         for opportunity in opportunities
         if opportunity.expected_giving_date == today
     ][0]
-    charge(opp)
+    try:
+        charge(opp)
+    except ChargeException:
+        # TODO should we alert slack? Did not because we had no notifications here before.
+        pass
```

Comment:
hmm I wonder why we don't send these even as a success. Maybe for privacy because it's more so paying for a product; rather than a donation? In any case I don't think it hurts to send the rejected ones like we do for the others - lets us know if something fishy is going on.

Reply:
I was thinking it might be for privacy too. I think it's probably ok, but just in case I'm going to add a question for Daniel. It's easy enough to add later once he confirms.